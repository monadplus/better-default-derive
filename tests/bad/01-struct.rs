#[derive(Debug, PartialEq, Eq, Default)]
struct B {
    a: u8,
    b: String,
}

#[derive(Debug, PartialEq, Eq, better_default_derive::Default)]
struct A {
    a: u8,
    b: B,
}

fn main() {}
