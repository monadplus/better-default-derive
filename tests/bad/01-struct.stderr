error: #[derive(Default)] is only supported for enums
 --> tests/bad/01-struct.rs:8:8
  |
8 | struct A {
  |        ^
